docker/stix2/Pipfile

```
@@ -0,0 +1,12 @@
+[[source]]
+name = "pypi"
+url = "https://pypi.org/simple"
+verify_ssl = true
+
+[dev-packages]
+
+[packages]
+stix2 = "*"
+
+[requires]
+python_version = "2.7"
```

```suggestion
python_version = "3.7"
```

Change to 3.7 then go into stix2 dir and run:
```
pipenv --rm
pipenv sync
```